src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -15,14 +15,15 @@ public class Drive extends SubsystemBase {
     private final DriveIO io;
     private driveTrainState mode;
     private final DriveIOInputsAutoLogged inputs = new DriveIOInputsAutoLogged();
-    private final SimpleMotorFeedforward driveff =
-            new SimpleMotorFeedforward(Constants.DriveConstants.kS, Constants.DriveConstants.kV);
+
+    private PIDController thetaController = new PIDController(0, 0, 0); // placeholders
```

Comment:
This will have to have non-zero gains in order to control the robot to line up with the target